x-pack/filebeat/module/zeek/signature/test/signature-json.log

```diff
@@ -0,0 +1 @@
+{"ts":1611852809.869245,"uid":"CbjAXE4CBxJ8W7VoJg","src_addr":"192.168.116.149","src_port":51617,"dst_addr":"192.168.116.172","dst_port":445,"note":"Signatures::Sensitive_Signature","sig_id":"my-second-sig","event_msg":"192.168.116.149: TCP traffic","sub_msg":""}
```

Comment:
Any possibility you could change `src_addr` and `dst_addr` to be public ips that are resolvable with geoip lookup? You can always just grab a public ip from another example log. If these are private ips we won't be able to fully see the pipeline at work in the test.

Reply:
done